test/sessionEventAndCoC.js

```diff
@@ -0,0 +1,273 @@
+      let sizesArr = [
+        [300, 600],
+        [720, 600]
+      ];
+      eventPage.getScrollbarVisibility(sizesArr).then(function(statusArr) {
+        eventPage.driver.manage().window().maximize();
+        assert.deepEqual(statusArr, [false, false]);
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking the title of the page', function(done) {
+      eventPage.getEventName().then(function(eventName) {
+        assert.equal(eventName, "FOSSASIA Summit 2017");
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking the presence of tweet section', function(done) {
+      eventPage.checkTweetSection().then(function() {
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Check whether the down button is working or not', function(done) {
+      eventPage.checkDownButton().then(function(offset) {
+        assert.equal(offset, 0);
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking the presence of Sponsors section', function(done) {
+      eventPage.checkSponsorSection().then(function() {
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking broken links in of Sponsors section', function(done) {
+      eventPage.getSponsorsBrokenLinks().then(function(brokenLinksCount) {
+        assert.equal(brokenLinksCount, 1);
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking the presence of ticket button', function(done) {
+      eventPage.checkTicketButton().then(function() {
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking the functionality of ticket button', function(done) {
+      eventPage.checkTicketFunctionality().then(function(brokenLinksCount) {
+        assert.equal(brokenLinksCount, 0);
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+    it('Checking the background colors for dark theme', function (done) {
+      eventPage.visit('http://localhost:5000/live/preview/a@a.com/MozillaAllHands2017/index.html');
+      eventPage.getBackgroundColor('event').then(function(bgcolorArr) {
+        assert.deepEqual(bgcolorArr, ['rgba(51, 61, 90, 1)', 'rgba(35, 41, 58, 1)']);
+        done();
+      }).catch(function(err) {
+        done(err);
+      });
+    });
+
+  });
+
+  describe('Testing Code of Conduct page', function(){
+    before(function() {
+      cocPage.init(driver);
+      cocPage.visit('http://localhost:5000/live/preview/a@a.com/FOSSASIASummit2017/CoC.html')
+    });
+
+    it('Checking the presence of Code of Conduct section', function(done) {
+      cocPage.checkCoCsection().then(function () {
+        done();
+      }).catch(function (err) {
+        done(err);
+      })
+    });
+
+  });
+});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)